### ISSUE TITLE
Test fails if decimal separator is set to "."

### ISSUE THREAD

**Issue by szarnyasg** [jexp/neo4j-shell-tools#64]
Hi!

The tests fail on `mvn clean install`. Maybe this is caused by the fact that the system's decimal separator is set to `.` instead of `,`? I tested it on Linux Mint 17.1 x64 en_US.

```
testToStringWithDouble(org.neo4j.shell.tools.imp.ImportCypherAppTest)  Time elapsed: 0.205 sec  <<< FAILURE!
org.junit.ComparisonFailure: expected:<"10000000[,]00"> but was:<"10000000[.]00">
    at org.junit.Assert.assertEquals(Assert.java:115)
    at org.junit.Assert.assertEquals(Assert.java:144)
    at org.neo4j.shell.tools.imp.ImportCypherAppTest.assertFile(ImportCypherAppTest.java:198)
    at org.neo4j.shell.tools.imp.ImportCypherAppTest.testToStringWithDouble(ImportCypherAppTest.java:190)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:497)
    at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
    at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
    at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
    at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
    at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
    at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
    at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
    at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
    at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
    at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
    at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
    at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
    at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
    at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
    at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:252)
    at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:141)
    at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:112)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:497)
    at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
    at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
    at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
    at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
    at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
```

Gabor

**Comment by mmichalowicz** [comment]
I have the same problem. I'll look at it but I'm hoping the authors get involved since the failed test may be indicative of a larger problem....  I am compiling 2.2 but 2.1 fails with yet a different exception....

Running Mac Osx Yosemite

**Comment by jexp** [comment]
Probably just a locale problem, my default locale is german which sets the decimal separator to comma.

**Comment by jexp** [comment]
Should be fixed in the 2.3 branch, please check.

**Comment by szarnyasg** [comment]
Yes, the tests work fine now. Thanks!
